### ISSUE TITLE
Keyboard is ignored temporarily until key U is pressed (v13.5.0)

### ISSUE THREAD

**Issue by mkaflowski** [pqrs-org/Karabiner-Elements#2756]
I get "Keyboard is ignored temporarily until key U is pressed" for Apple Internal Keyboard when I connect/ disconnect monitor or run Kodi on v13.5.0. This is not present at v13.4.0.

**Comment by MarkKahn** [comment]
This is one of the most offensive things I've ever seen any program do, ever.  Seriously, "Here's an annoying popup that you have to read and then hunt the key, sometime several keys in a row".  Whoever wrote this must have been smoking crack.  I'm sure there's some obscure edge case where this has value, but there have been a dozen issues on this repo about this and they've all just been closed without any resolution.

**Comment by sloanlance** [comment]
I'm getting a similar error.  Sometimes when I put my MacBook Pro to sleep, it shows this message when I wake it up…

> Apple Internal Keyboard / Trackpad is ignored temporarily until {"key_code":"right_shift"} is pressed again.

See the following screen shot for context…

<img width="444" alt="Screen Shot 2021-10-21 at 13 17 33" src="https://user-images.githubusercontent.com/17595351/140798972-616e0381-09e5-4ff7-bd52-529ee667281d.png">

However, it doesn't seem to get in the way of anything I do with my keyboard/trackpad.  Even the Karabiner bindings I've set seem to work.

Finally, I can press the key it specifies to make it go away immediately, but also sometimes it just goes away on its own.

I'm using…
* macOS Version 11.6 (20G165)
* Karabiner-Elements 13.7.0

**Comment by MarkKahn** [comment]
bump.  bad bot

**Comment by Sinled** [comment]
Is there any fix for this? Half of the time karabiner requests `scan_next_track` button, that doesn't exist on my trackball, and only replug helps, it is really frustrating =(

**Comment by clemsam** [comment]
Hi, Duckduck-googled and found at
https://karabiner-elements.pqrs.org/docs/manual/misc/ignored-temporarily/:

About "Device is ignored temporarily" alert
“Device is ignored temporarily” alert will be shown in any of the following cases:

––  If you hold keys or buttons down before Karabiner-Elements open the device.

––  Your device sends unintented key up events regardless you do not touch the keys.
     (e.g., Swiftpoint ProPoint sends key up events of the music play key periodically.)

Hope this may help?!?

**Comment by steerio** [comment]
No, it doesn't help. It happens a dozen times daily, and I don't have any external keyboards or pointing devices. It's extremely annoying, there've been issues about it since at least 2019, and no "update to beta version" has ever solved it.
